src/EventSubscriber/SetClientServiceUuid.php

```diff
@@ -21,6 +19,13 @@
 class SetClientServiceUuid implements EventSubscriberInterface {
 
   use PlatformCmdOutputFormatterTrait;
+  use PlatformGroupTrait;
+  use PlatformSitesTrait;
+
+  public const GROUP_CONFIG_LOCATION = [
+    '.commonconsole',
+    'groups',
+  ];
```

Comment:
I couldn't find any place where this constant is in use. Also there is code in contenthub-console-helpers, in PlatformGrouTrait:getGroupFilePath() where you redefine the config location. We should use the constant then.